server/StudySharp.ApplicationServices/Commands/UpdatePracticalBlockCommand.cs

```diff
@@ -0,0 +1,33 @@
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using StudySharp.Domain.Constants;
+using StudySharp.Domain.General;
+using StudySharp.Domain.Models;
+using StudySharp.DomainServices;
+
+namespace StudySharp.ApplicationServices.Commands
+{
+    public sealed class UpdatePracticalBlockCommand : IRequest<OperationResult>
+    {
+        public int Id { get; set; }
+    }
+
+    public sealed class UpdatePracticalBlockCommandHandler : IRequestHandler<UpdatePracticalBlockCommand, OperationResult>
+    {
+        private readonly StudySharpDbContext _studySharpDbContext;
+
+        public async Task<OperationResult> Handle(UpdatePracticalBlockCommand request, CancellationToken cancellationToken)
+        {
+            var practicalBlock = await _studySharpDbContext.PracticalBlocks.FindAsync(request.Id, cancellationToken);
```

Comment:
You're retrieving practical block without any changes, do not apply anything, and trying to update it. Sorry, what?